app/models/photo.rb

```diff
@@ -11,11 +11,19 @@ class Photo < ActiveRecord::Base
 
   def self.search(params, rover)
     photos = search_by_date params
+
     if params[:camera]
       if photos.any?
         photos = photos.search_by_camera params, rover
       end
     end
+
+    photos = photos.order(:camera_id, :id)
+
+    if params[:page]
+      photos = photos.page(params[:page]).per params[:per_page]
```

Comment:
It isn't super important in this small app, but generally things like pagination are best kept in the controller, as they are more strictly related to presentation. Pagination isn't really a part of searching, and you can imagine if there were other reasons to search for photos, we probably wouldn't want pagination.

Reply:
That makes sense! I moved this logic back to PhotoHelper, so it can be shared between photos_controller and latest_photos_controller